php_phongo.c

```diff
@@ -1289,6 +1289,7 @@ static bool php_phongo_apply_wc_options_to_uri(mongoc_uri_t *uri, bson_t *option
 	return true;
 } /* }}} */
 
+#ifdef MONGOC_ENABLE_SSL
```

Comment:
I can't see where the php_phongo_fetch_ssl_opt_string function is used though, so please check and make sure it is only called when #ifdef MONGOC_ENABLE_SSL is true as well.